lib/src/working_copy.rs

```diff
@@ -27,6 +26,7 @@ use std::os::unix::fs::PermissionsExt;
 use std::path::{Path, PathBuf};
 use std::sync::Arc;
 use std::time::UNIX_EPOCH;
+use std::fs;
```

Comment:
? Why not sorted? I mean, yes, `:` comes before `;`, but at worst I would expect that to produce:

```
use std::fs::{File, Metadata, OpenOptions};
use std::fs;
```

Reply:
(The next commit undoes this, I think?)

Reply:
I don't know what happened here, but I've fixed it now.